Grids: Add onKeyDown Drop-Down Editor Note (#8716)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/1 Configuration/onKeyDown.md

    }
    export default App;

---

> [note] When a drop-down editor within {WidgetName} is in focus, **onKeyDown** is not fired for up/down arrow key presses.

Reviewer: ```suggestion
[note] note] When a drop-down editor in the {WidgetName} has focus,  **onKeyDown** does not fire for up and down arrow key presses
```
